index.html

```diff
@@ -107,6 +107,35 @@ <h2>A project for first-time contributions</h2>
         <!-- ============== ^^^^ TEMPLATE ^^^^ ============== -->
         <!-- ============================================================== -->
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
+
+        <div class="card">
+          <p class="name">Dwight</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="www.twitter.com" target="_blank">@dwight_donis</a>
+          </p>
+          <p class="about">Self-taught Web Dev trying to get comfortable with Github </p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://www.w3schools.com" target="_blank" title="First Resource">
+                  W3Schools
+                </a>
+              </li>
+              <li>
+                <a href="https://www.youtube.com" target="_blank" title="Second Resource">
+                  YouTube
+                </a>
+              </li>
+              <li>
+                <a href="https://www.codecademy.com" target="_blank" title="Third resource">
+                  Codeacademy
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
```

Comment:
add `<!-- ________ *username* Contributor card END ________ -->` after the end of your card